website/content/docs/updates/release-notes.mdx

```diff
@@ -26,218 +63,302 @@ Deprecated in 1.20.x | Retired in 1.20.x
-      Pending
+      Configure default and back up login methods for Vault GUI to reduce
+      complexity and confusion.
       <br /><br />
-      Learn more: <a href="/vault/docs/">TDB</a>
+      Learn more: <a href="/vault/docs/ui/custom-login">Manage custom login settings</a>
     </td>
   </tr>
-
-  </tbody>
-</table>
-
-
-## Manage identities and authentication
-
-@include 'release-notes/section-notes/identity.mdx'
-
-<table>
-  <thead>
-    <tr>
-      <th style={{verticalAlign: 'middle'}}>Release</th>
-      <th style={{verticalAlign: 'middle'}}>Update</th>
-      <th style={{verticalAlign: 'middle'}}>Description</th>
-    </tr>
-  </thead>
-
-  <tbody>
-
   <tr>
     <td style={{verticalAlign: 'middle'}}>
-      Pending
+      Client count dashboard updates
     </td>
-    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Pending</td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Enhanced</td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Enterprise</td>
     <td style={{verticalAlign: 'middle'}}>
-      Pending
+      Provides improved visibility into client count attribution, increases
+      accuracy by removing estimates, and sets the current billing period in
+      Vault GUI based on the current Vault configuration.      
       <br /><br />
-      Learn more: <a href="/vault/docs/">TDB</a>
+      Learn more: <a href="/vault/docs/concepts/client-count/client-usage">Client usage</a> overview
     </td>
   </tr>
-
-  </tbody>
-</table>
-
-
-## Manage static secrets
-
-@include 'release-notes/section-notes/static-secrets.mdx'
-
-<table>
-  <thead>
-    <tr>
-      <th style={{verticalAlign: 'middle'}}>Release</th>
-      <th style={{verticalAlign: 'middle'}}>Update</th>
-      <th style={{verticalAlign: 'middle'}}>Description</th>
-    </tr>
-  </thead>
-
-  <tbody>
-
   <tr>
     <td style={{verticalAlign: 'middle'}}>
-      Pending
+      Client count current month accuracy
     </td>
-    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Pending</td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Enhanced</td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Enterprise</td>
     <td style={{verticalAlign: 'middle'}}>
-      Pending
+      Removed partial month estimates from client count to improve client count 
+      accuracy for the current month.
       <br /><br />
-      Learn more: <a href="/vault/docs/">TDB</a>
+      Learn more: <a href="/vault/api-docs/system/internal-counters#partial-month-client-count">Partial month client count endpoint</a>
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      GUI Namespace picker updates
+    </td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Enhanced</td>
+    <td style={{verticalAlign: 'middle', textAlign: 'center'}}>Enterprise</td>
+    <td style={{verticalAlign: 'middle'}}>
+      Search, filter and navigate to namespaces in the GUI without having to
```

Comment:
@finnstech Tweaked slightly from your suggestion since "Also reduced performance load and enhanced accessibility" isn't a complete sentence. Please take a look and let me know if you'd like additional changes